CONTRIBUTING.md

```diff
@@ -16,34 +17,25 @@ Before reporting an issue, see the following wiki articles:
 - [Troubleshooting][wiki-troubleshooting]
 - [Frequently asked questions][wiki-faq]
 
-If your issue isn't mentioned there:
-
-- Verify that it hasn't already been reported.
-- If not already running the latest version of Neovim, update to it to see if
-  your problem persists.
-- If you're experiencing compile or runtime warnings/failures, try searching for
-  the error message(s) you received (if any) on [Neovim's issue tracker][github-issues].
-    - For build issues, see
-      [Troubleshooting#build-issues][wiki-troubleshooting-build-issues].
-    - For runtime issues, see
-      [Troubleshooting#runtime-issues][wiki-troubleshooting-runtime-issues].
-      If your issue isn't mentioned there, try reproducing it using `nvim`
-      with the smallest possible `vimrc` (or none at all via `nvim -u NONE`),
-      to rule out bugs in plugins you're using.
-      If you're using a plugin manager, comment out your plugins, then add
-      them back in one by one.
+If your issue isn't mentioned there, verify the following:
+
+- You can reproduce it on the latest version of Neovim.
+- That it hasn't already been reported on [Neovim's issue tracker][github-issues];
+  make sure to search for any error messages you may have encountered.
+- For runtime issues, try to reproduce them using `nvim -u NONE`. If that
+  fails then try `nvim -u NORC` instead. If neither of those work then your
+  issue is probably related to a plugin, so try reproducing it using a minimal
+  `vimrc` (see [#2458](https://github.com/neovim/neovim/issues/2458)).
```

Comment:
I would suggest doing the opposite:
- Try `-u NORC` to see if the issue is in the user's vimrc.
- If that fails, `-u NONE` to see it the problem is in the runtime.

Not sure where trying out the minimal vimrc fits in this.